File: DSA/Arrays/subArrays/maximum-subarray-easy/python/maximumSubArrayEasy.py
def maxSubarray(self, A, B, C):
        maxSum = 0
        for i in range(A):
            sum = 0
            for j in range(i, A):
                sum +=C[j]
                if sum > maxSum and sum <= B:
                    maxSum = sum
        
        return maxSum

File: DSA/Arrays/subArrays/maximum-subarray-easy/javascript/maximumSubArrayEasy.js
export const maxSubarray = (A, B, C) => {
  let maxSum = 0n
  for (let i = 0; i < A; i++){
      let sum = 0n
      for (let j = i; j < A; j++){
          sum += C[j]
          if (sum > maxSum && sum <=B) {
              maxSum = sum
          }
      }
  }
  return Number(maxSum)
}